Program.cs

```diff
@@ -4,6 +4,10 @@ class Program
 {
     static void Main(string[] args)
     {
-        Console.WriteLine("Hello, world!");
+        Console.Write("What is your first name? ");
+        string firstName = Console.ReadLine();
+        Console.Write("What is your last name? ");
+        string lastName = Console.ReadLine();
+        Console.WriteLine($"Hello, {firstName} {lastName}");
```

Comment:
May want to consider internationalization where surname and given name order swaps.